tests/wpt/meta/css/css-grid/masonry/tentative/grid-placement/masonry-grid-placement-named-lines-001.html.ini

```diff
@@ -1,2 +1,2 @@
 [masonry-grid-placement-named-lines-001.html]
-  expected: FAIL
+  expected: CRASH
```

Comment:
Why is this crashing?

Reply:
I'm not sure on the specifics, but the high-level issue here is that we don't support masonry (or named lines). There are 4 or 5 masonry WPT crashing (this is just only one that's new). Masonry is essentially an entirely separate layout mode to regular grid, so until we get around to implementing we probably ought to consider disabling (runtime feature flagging) it in Stylo.

Reply:
Masonry is already disabled for Servo: https://github.com/servo/stylo/blob/03ffc85c45f53a3b11994f7c14adce7b90aa9c34/style/values/specified/grid.rs#L305-L309

I prefer not to introduce new crashes without investigation.

Reply:
I've tracked down the crash and fixed it.